### PR TITLE
CLOUDP-213199 Atlas Go SDK docs changes for binary responses

### PR DESCRIPTION
**Pull request by fmenezes**
## Description

Atlas Go SDK docs changes for binary responses

Link to any related issue(s): https://jira.mongodb.org/browse/CLOUDP-213199

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: docs/doc_5_best_practices.md [hunk 1]
```diff
@@ -58,11 +58,13 @@ When you have confirmed that the `time.Time pointer` is non-nil, you can safely
 
 In the Atlas Go SDK, the `io.ReadCloser` type is used to return binary data from APIs. 
 
-1. Use `io.Copy` if the intention is to store on a file or pass through another steam
+- Use `io.Copy` if the intention is to store on a file or pass through another steam
```
Comment (kanchana-mongodb) [inline docs/doc_5_best_practices.md:61]:
```suggestion
- Use `io.Copy` to store on a file or pass through another steam.
```

File: docs/doc_5_best_practices.md [hunk 2]
```diff
@@ -58,11 +58,13 @@ When you have confirmed that the `time.Time pointer` is non-nil, you can safely
...
 
-2. Use `io.ReadAll` if the intention is to read all bytes in memory
+- Use `io.ReadAll` if the intention is to read all bytes in memory
```
Comment (kanchana-mongodb) [inline docs/doc_5_best_practices.md:63]:
```suggestion
- Use `io.ReadAll` to read all bytes in memory.
```

File: docs/doc_5_best_practices.md [hunk 3]
```diff
@@ -58,11 +58,13 @@ When you have confirmed that the `time.Time pointer` is non-nil, you can safely
...
 
-3. Call the `.Close()` function after done reading the data
+- Call the `.Close()` function after done reading the data
```
Comment (kanchana-mongodb) [inline docs/doc_5_best_practices.md:65]:
```suggestion
- Call the `.Close()` function after reading the data.
```

File: docs/doc_5_best_practices.md [hunk 4]
```diff
@@ -58,11 +58,13 @@ When you have confirmed that the `time.Time pointer` is non-nil, you can safely
 
 In the Atlas Go SDK, the `io.ReadCloser` type is used to return binary data from APIs.
```
Comment (kanchana-mongodb) [inline docs/doc_5_best_practices.md:59]:
```suggestion
In the Atlas Go SDK, the `io.ReadCloser` type can be used to return binary data using the APIs. 
```

Comment (fmenezes) [reply docs/doc_5_best_practices.md:59]: @kanchana-mongodb this is not a "can be" the code is auto-generated it will always be `io.ReadCloser`

**Review by kanchana-mongodb — APPROVED** [review]
LGTM

**Pull request merged**